fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/WorldRenderEvents.java

```diff
@@ -279,13 +279,15 @@ public interface BeforeBlockOutline {
 		 * Event signature for {@link WorldRenderEvents#BEFORE_BLOCK_OUTLINE}.
 		 *
 		 * @param context  Access to state and parameters available during world rendering.
+		 * @param translucent If {@code true}, current block outline is being rendered after translucent terrain.
+		 *                    Otherwise, it is being rendered after solid terrain.
 		 * @param hitResult The game object currently under the crosshair target.
 		 * Normally equivalent to {@link MinecraftClient#crosshairTarget}. Provided for convenience.
 		 * @return true if vanilla block outline rendering should happen.
 		 * Returning false prevents {@link WorldRenderEvents#BLOCK_OUTLINE} from invoking
 		 * and also skips the vanilla block outline render, but has no effect on other subscribers to this event.
 		 */
-		boolean beforeBlockOutline(WorldRenderContext context, @Nullable HitResult hitResult);
+		boolean beforeBlockOutline(WorldRenderContext context, boolean translucent, @Nullable HitResult hitResult);
```

Comment:
Sorry, I missed this earlierthis is also a breaking change. You could you maybe pass this in via the context (not ideal but at least is not breaking), otherwise it needs a whole new event.

Reply:
Or another method in the interface delegating to the old one by default, but implementing it would be clunky.

Reply:
I think the easy option is to add it to the context as `translucentBlockOutline`, and maybe document its designed for this event.